components/Paper/Tabs/DiscussionTab.js

```diff
@@ -196,7 +196,9 @@ const DiscussionTab = (props) => {
                       key={`thread-${t.data.id}`}
                       {...t.data}
                       bounties={
-                        bountyMap[t.data.id]
+                        props.bounties && props.bounties.length
```

Comment:
This needs to be updated. Kind of messy right now, but we have two bounty types:

1. Questions
2. Comments

We're using one field right now "bounties" for both of them. props.bounties is from Questions and the bountyMap and t.data.bounties is for comment bounties